src/main/java/info/u_team/useful_railroads/data/provider/UsefulRailroadsItemTagsProvider.java

```diff
@@ -18,6 +18,7 @@ public UsefulRailroadsItemTagsProvider(GenerationData data) {
 	protected void registerTags() {
 		copy(BlockTags.RAILS, ItemTags.RAILS);
 		getBuilder(TRACK_BUILDER_RAILS).add(Items.POWERED_RAIL, UsefulRailroadsBlocks.HIGHSPEED_RAIL.asItem());
+		getBuilder(TRACK_BUILDER_RAILS).add(Items.POWERED_RAIL, UsefulRailroadsBlocks.SPEED_CLAMP_RAIL.asItem());
```

Comment:
This will add the ```Items.POWERED_RAIL``` twice to the TRACK_BUILDER_RAILS tag. The line should be:
```java
getBuilder(TRACK_BUILDER_RAILS).add(Items.POWERED_RAIL, UsefulRailroadsBlocks.HIGHSPEED_RAIL.asItem(), UsefulRailroadsBlocks.SPEED_CLAMP_RAIL.asItem());
```